node/cn/tracers/testdata/call_tracer_selfdestruct.json

```diff
@@ -57,7 +57,10 @@
   "result": {
     "calls": [
       {
-        "type": "SELFDESTRUCT"
+        "type": "SELFDESTRUCT",
+        "from": "0xf0b1b2a91af3b0a0a5389ea80bffdc42cf86b7e3",
+        "to": "0xf0b1b2a91af3b0a0a5389ea80bffdc42cf86b7e3",
+        "value": "0x0"
```

Comment:
Is this okay without printing `gasIn` and `gasCost`?

Reply:
@jeongkyun-oh 
Yeap. 
`gasIn` and `gasCost` are used when [calculating `gasUsed` field of call struct](https://github.com/klaytn/klaytn/blob/dev/node/cn/tracers/internal/tracers/call_tracer.js#L164).
And both fields are not contained in the return object. You can check [this out in call_tracer.js](https://github.com/klaytn/klaytn/blob/0241baf093d0bd24217ef21664691d8b48c57461/node/cn/tracers/internal/tracers/call_tracer.js#L299-L325).
As you can see, there are no fields `gasIn` and `gasCost`.

And also the output of Ethereum's JS Call tracer after this patch is same with Klaytn.
So there will be no issue for this.